scripts/jenkins/Jenkinsfile

```diff
@@ -902,6 +923,49 @@ def buildHomepage() {
   return homepageTasks
 }
 
+def deployWebUI() {
+  node("elektrad") {
+    docker.withRegistry("https://${REGISTRY}",
+                        'docker-hub-elektra-jenkins') {
+      def backendName = "elektrad"
+      def frontendName = "webd"
+      def backend = docker.image(DOCKER_IMAGES.webui_elektrad.id)
+      def frontend = docker.image(DOCKER_IMAGES.webui_webd.id)
+      backend.pull()
+      frontend.pull()
+
+      sh "docker stop -t 5 ${backendName} || /bin/true"
```

Comment:
this functionality could be moved into a function as it is now used 4 times in a row